src/main/groovy/org/mockito/release/internal/gradle/BaseJavaLibraryPlugin.java

```diff
@@ -90,5 +91,9 @@ public void execute(MavenPublication publication) {
 
         //so that we flesh out problems with maven publication during the build process
         project.getTasks().getByName("build").dependsOn("publishToMavenLocal");
```

Comment:
I think this question is unrelated @mstachniuk PR. The plugins are opinionated and they use new publishing plugin. If someone wants to use classic upload archives then he would have to avoid using our plugins and wire our tasks himself. That's not very convenient. If we have use cases / users asking for supporting classic publication, we can add it. Perhaps you can contribute? :)